pkg/plugin/plugin_test.go

```diff
@@ -179,3 +180,38 @@ func TestRotateClusterSecrets(t *testing.T) {
 		t.Errorf("plugin.RotateClusterSecrets error = %v", err)
 	}
 }
+
+func TestBackupEtcdCluster(t *testing.T) {
+	mockCtrl := gomock.NewController(t)
+	defer mockCtrl.Finish()
+
+	backupName := "etcd-backup"
+	cs := &api.OpenShiftManagedCluster{
+		Config: api.Config{
+			Images: api.ImageConfig{
+				EtcdBackup: "quay.io/openshift-on-azure/etcdbackup:latest",
```

Comment:
will this need a version at the end (instead of latest)?

Reply:
since this is just a unit test, I think latest is ok